### PR TITLE
stacks: add metadata for Rook Stack resources

### PR DESCRIPTION
**Pull request by displague**
### Description of your changes

This PR adds metadata and resources for all CRDs in this Rook stack.
Each CRD should have an icon file, ui-schema.yaml, and resource information such as title, overview, readme, etc.

Icons attained through:
* https://worldvectorlogo.com/logo/cockroachdb
* https://github.com/yugabyte/yugabyte-db/blob/master/managed/ui/src/components/common/YBLogo/images/yb_ybsymbol_original.svg

This PR includes links to the **eventual** Crossplane API documentation URLs for these Stack resource.

Some automation was used to create these files:

```sh
alias yaml2json="python -c 'import sys, yaml, json; y=yaml.load(sys.stdin.read()); print(json.dumps(y))'"
for file in ../../../crd/*yaml ; do
  desc=$(cat $file | yaml2json | jq '.spec.validation.openAPIV3Schema.description' );
  kind=$(cat $file | yaml2json | jq '.spec.names.kind' | tr A-Z a-z | tr -d \" ) ;
  echo -e "id: $kind\ntitle: \ntitlePlural: \ncategory: \noverviewShort: $desc\noverview: |\n \nreadme: |\n \n" > $kind.resource.yaml;
  touch $kind.icon.svg;
done
```

```sh
for resource in *resource.yaml; do 
  echo -e "version: 0.4\nconfigSections: []\n\nresourceConfitionPaths:\n- .status.conditionedStatus.conditions[?(@.type=='Ready')]" > ${resource/resource/ui-schema};
done
```

### Checklist

I have:
- [ ] Run `make reviewable` to ensure this PR is ready for review.
- [ ] Ensured this PR contains a neat, self documenting set of commits.
- [ ] Updated any relevant [documentation], [examples], or [release notes].
- [ ] Updated the RBAC permissions in [`clusterrole.yaml`] to include any new types.

[documentation]: https://github.com/crossplaneio/crossplane/tree/master/docs
[examples]: https://github.com/crossplaneio/crossplane/tree/master/cluster/examples
[release notes]: https://github.com/crossplaneio/crossplane/tree/master/PendingReleaseNotes.md
[`clusterrole.yaml`]: https://github.com/crossplaneio/crossplane/blob/master/cluster/charts/crossplane/templates/clusterrole.yaml

### CODE REVIEW

File: config/stack/manifests/resources/cockroachclusterclass.ui-schema.yaml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+version: 0.4
+configSections: []
+
+resourceConditionPaths:
+- .status.conditionedStatus.conditions[?(@.type=='Ready')]
```
Comment (jbw976) [inline config/stack/manifests/resources/cockroachclusterclass.ui-schema.yaml]: same comment as in the stack-gcp PR, I don't think classes have any status or conditions. this applies to the yugabyte class too.

File: config/stack/manifests/resources/yugabytecluster.resource.yaml [hunk 2]
```diff
@@ -0,0 +1,23 @@
+id: yugabytecluster
+title: YugabyteDB Cluster
+titlePlural: YugabyteDB Clusters
+category: Database
+overviewShort: "A YugabyteCluster configures a Rook 'YugabyteCluster'"
```
Comment (jbw976) [inline config/stack/manifests/resources/yugabytecluster.resource.yaml]: according to https://rook.io/docs/rook/v1.1/yugabytedb-cluster-crd.html, the Rook CRD kind is actually `YBCluster`.  If we want to use the fully qualified name of the type, it's `ybclusters.yugabytedb.rook.io`

File: config/stack/manifests/resources/cockroachcluster.resource.yaml [hunk 3]
```diff
@@ -0,0 +1,35 @@
+id: cockroachcluster
+title: CockroachDB Cluster
+titlePlural: CockroachDB Clusters
+category: Database
+overviewShort: "A CockroachCluster configures a Rook 'CockroachCluster'"
```
Comment (jbw976) [inline config/stack/manifests/resources/cockroachcluster.resource.yaml]: According to https://rook.io/docs/rook/v1.1/cockroachdb-cluster-crd.html, the actual kind of the Rook cockroach CRD is `Cluster`.  If we want to use the fully qualified type it's `clusters.cockroachdb.rook.io`

**Comment by displague** [comment]
The stack installs fine and the Yugabyte and Cockroach CRDs reflect the updated ui-schema.yaml data for each CRD:

```yaml
    stacks.crossplane.io/ui-schema: |-
      version: 0.4
      configSections: []
      resourceConditionPaths: []
```

**Pull request merged**